thunder/tests/test_interpreter.py

```diff
@@ -363,7 +363,7 @@ def fn1(a, b):
 
     # test order for collisions
     def fn2(a, b):
-        return {"a": a, "a": b}
+        return {"a": b}
```

Comment:
I'm not quite sure about this change given the comment of "test order for collisions" above and I'm speculating noqa might be better